Slim template

clarksandholtz opened the pull request:
## Summary

Currently, there is only a template that includes several models and a bunch of example code. This template gets you all the necessary dependencies without adding anything extra so you can plug in only the necessary models, dependencies, and source code.

## Changelog

Duplicated the original template and made the following changes:
- Removed the models
- Removed 3rd party dependencies not needed to run a model with PTL
- Simplified `src` folder to contain 1 file called `App.tsx` and replaced it's contents with what used to be in `MyDemos.tsx` modified to make more sense.

## Test Plan

Tested template with `npx react-native init Ejemplo --template file:///Users/clarksandholtz/src/live/react-native-template-ptl` and was able to run the app.

chrisklaiber commented:
Thanks, @clarksandholtz! Having a slim template will be nice, especially for experienced PyTorch Live users.

I wonder if we should change the the folder path. The current folder, `react-native-template-ptl`, is very close to the existing  `react-native-template-pytorch-live` and is semantically identical. How about react-native-template-pytorch-live-base` or `-slim` or `-simple`?

Also, is there a file-synchronization linter? For a lot of the boilerplate files, which appears to be almost all of the files, I imagine we want the slim and non-slim template to have identical file contents. It would be awesome if a linter could point to mismatches!

raedle commented on react-native-template-ptl/package.json:
```diff
@@ -0,0 +1,8 @@
+  "version": "0.0.0",
+  "description": "A minimal React Native template for PyTorch Live",
+  "repository": "",
+  "author": "Facebook",
```
```suggestion
  "author": "Meta Platforms, Inc.",
```

raedle commented on react-native-template-ptl/template.config.js:
```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```
Let's change all license headers to match the new license header for Meta Platforms, Inc.

```suggestion
 * Copyright (c) Meta Platforms, Inc. and affiliates.
```

raedle commented on react-native-template-ptl/template/android/app/src/main/java/org/pytorch/live/example/BertQAPacker.java:
```diff
@@ -0,0 +1,102 @@
+/*
```
This file is not needed in the slim template :)

raedle commented on react-native-template-ptl/template/package.json:
```diff
@@ -0,0 +1,47 @@
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "start-reset-cache": "yarn push-core-package && react-native start --reset-cache",
```
The `push-core-package` script doesn't exist. Should this be removed?

raedle reviewed (changes requested):
Overall looks great! A couple of small changes like removing unused Java packer class and updating license headers :)

clarksandholtz commented:
@raedle thanks for finding those things to change! I removed the BertQA packer file and the extra command in the `start-reset-cache` command. I also changed all the names from Facebook to Meta Platforms.

raedle reviewed (approved):
Looks great! Thanks for the changes!
